### ISSUE TITLE
Commands don't update on permission update

### ISSUE THREAD

**Issue by hapily04** [LooFifteen/LuckPerms#6]
### Description

Commands do not resend upon permission change, so they're messed up in tab completions.

### Reproduction Steps

1. Have a dummy command with a condition that checks if the player has a certain permission:
```
        setCondition((sender, _) -> sender instanceof ConsoleSender || (sender instanceof LuckPermsPlayer player && player.hasPermission(permissionNode)));
````
2. Give the player the permission via `/luckperms user username permission set permissionnode true`
3. Type out the command in chat and notice you won't see it pop up like a command until you rejoin.

### Expected Behaviour

Commands should be refreshed upon permission change

### Server Details

Minestom

### LuckPerms Version

5.4-SNAPSHOT

**Comment by LooFifteen** [comment]
Should be fixed in 2c965ab8c48ce23c0a325e0e23a075f5d9ae6cd0.

**Comment by hapily04** [comment]
Fixed it 🎊
